src/goto-analyzer/goto_analyzer_parse_options.cpp

```diff
@@ -555,8 +555,8 @@ int goto_analyzer_parse_optionst::perform_analysis(const optionst &options)
     return CPROVER_EXIT_SUCCESS;
   }
 
-  if(set_properties())
-    return CPROVER_EXIT_SET_PROPERTIES_FAILED;
+  if(cmdline.isset("property"))
+    ::set_properties(goto_model, cmdline.get_values("property"));
```

Comment:
What does this change mean?

Reply:
After removing the `try` block, `set_properties` only contained the two lines I've introduced and then `return false` so I inlined them.